azkaban-common/src/main/java/azkaban/project/FlowTriggerBean.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package azkaban.project;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Java bean loaded from YAML file to represent a flow trigger.
+ */
+public class FlowTriggerBean {
+
+  private long maxWaitMins;
+  private Map<String, String> schedule;
```

Comment:
why map?

Reply:
Because there are two key-value pairs for schedule in YAML file: 
  schedule:                    
    type: cron
    value: 0 0 1 ? * *